doc/source/admin/special_topics/performance_tracking.rst

```diff
@@ -5,34 +5,29 @@ Tracking performance over time and identifying slow queries in your Galaxy can b
 
 Most performance tracking requires sending metrics to a metrics collection server such as `Graphite <http://graphiteapp.org/>`__ or `StatsD <https://github.com/etsy/statsd/>`__. This document assumes you have already deployed a metrics server.
 
-uWSGI
+Gunicorn
 -----
 
-There is some built-in uWSGI support for performance logging. You can send uWSGI's internal metrics to a carbon (Graphite) server by setting the carbon option in your galaxy.yml:
+There is some built-in Gunicorn support for performance logging. You can send Gunicorn's internal metrics to a StatsD server by setting the `--statsd-host` and `--statsd-prefix` command line options for Gunicorn in the `gravity` section of `galaxy.yml`:
 
 .. code-block:: yaml
 
-   uwsgi:
-      socket: ...
-      carbon: 127.0.0.1:2003
+   gravity:
+      ...
+      gunicorn:
+        gunicorn_extra_args: `--statsd-host 127.0.0.1:8125 --statsd-prefix=galaxy`
+      ...
 
-Or a StatsD server via:
-
-.. code-block:: yaml
-
-   uwsgi:
-      socket: ...
-      stats-push: statsd:127.0.0.1:8125
```

Comment:
stats-push pushes to an external endpoint that collects statsd values (here we use telegraf, and it listens on that port, and collects things pushed by galaxy.) This is the most normal mode of operation.